modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/docMacros.ftl

```diff
@@ -24,6 +24,14 @@
   <input name="fulltext" id="filter-box" type="search" placeholder="Filter ${name}" class="searchFilter" autofocus />
 </#macro>
 
+<#macro javadoc class addLabel url>
+  <#if addLabel>Javadoc: </#if>
+  <#if url??>
+    <a href="${url}" target="_blank" class="javadoc">${url}-${class}</a>
```

Comment:
oups left a debug print here (there is a ftest on this), will fix it and relaunch

Reply:
--> https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-atchertchian-master-2/91/